File: Platazi_Learn/POO/Python/Car.py
from Account import Account

class Car:
    id          = int
    license     = str
    driver      = Account(str, str)
    passenger   = int

    def __init__(self, license, driver):
        self.license = license
        self.driver = driver

    def printDataCar(self):
        print('License: ', self.license)
        print('Driver Data: ', self.driver)
        print('Number of passenger: ', self.passenger)

    

File: Platazi_Learn/POO/JS/Car.js
class Car { 
    constructor(license, driver) {
        this.id;
        this.license = license;                       // string
        this.driver = driver;
        this.passenger;
    }
    printDataCar = () => {
        console.log('\n' + 'License: ' + this.license);
        console.log('\n' + 'Driver Data : ');
        console.log(this.driver);
    } 
}

export default Car;